Components/ProductToShop.php

```diff
@@ -1206,4 +1209,97 @@ public function makeMainVariant($shopId, $sourceId, $groupId)
         $this->manager->persist($article);
         $this->manager->flush();
     }
+
+    /**
+     * Updates the status of an Order
+     *
+     * @param string $localOrderId
+     * @param string $orderStatus
+     * @param string $trackingNumber
+     * @return void
+     */
+    public function updateOrderStatus($localOrderId, $orderStatus, $trackingNumber)
+    {
+        if ($this->config->getConfig('updateOrderStatus') == 1) {
+            $this->updateDeliveryStatus($localOrderId, $orderStatus);
+        }
+
+        if ($trackingNumber) {
+            $this->updateTrackingNumber($localOrderId, $trackingNumber);
+        }
+    }
+
+    /**
+     * @param $localOrderId
+     * @param $orderStatus
+     */
+    private function updateDeliveryStatus($localOrderId, $orderStatus)
+    {
+        $status = false;
+        if ($orderStatus === OrderStatus::STATE_IN_PROCESS) {
+            $status = Status::ORDER_STATE_PARTIALLY_DELIVERED;
```

Comment:
this is because we have different status in SDK and shopware this is  mapping we don't have a partially delivered status in sdk because the transaction in sdk are one to one transaction